tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -2868,6 +2898,10 @@ <h2>Show similarity to selected datapoint</h2>
         return stats && Object.keys(stats).length > 0;
       },
 
+      shouldHideCounterfactualModelSelector_: function(modelNames) {
```

Comment:
how about being dependent on "numModels" instead? Then just need to check if that num != 1.

Reply:
I wanted to make sure that parsedModelNames is populated before creating the window element, in case we end up in a state where numModels is set but parsedModelNames is still being processed. Right now it seems like parsedModelNames is updated before numModels, so again it may cause some inconsistencies (e.g. numModels=1 but parsedModelNames,length=2).